include/libtorrent/netlink.hpp

```diff
@@ -63,7 +63,7 @@ namespace libtorrent
 			std::memset(&m_sockaddr, 0, sizeof(sockaddr_nl));
 			m_sockaddr.nl_family = AF_NETLINK;
 			m_sockaddr.nl_groups = group;
-			m_sockaddr.nl_pid = pid;
+			m_sockaddr.nl_pid = std::uint32_t(pid);
```

Comment:
would it be safer to cast to ``pid_t``? I would be worried about a system that uses 64 bit numbers for pids.